src/ansi-c/c_qualifiers.cpp

```diff
@@ -9,6 +9,27 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "c_qualifiers.h"
 
 #include <ostream>
+#include <util/make_unique.h>
+
+c_qualifierst &c_qualifierst::operator=(const c_qualifierst &other)
```

Comment:
Why would you even want to do this?

Reply:
To implement `clone` in derived classes we create an instance of the derived object, use this assignment to copy the base class data members to the new instance and then copy the derived class data.